bedrock/firefox/templates/firefox/welcome/page17/page17-b.html

```diff
@@ -80,7 +80,7 @@ <h1 class="c-main-title">{{ main_title }}</h1>
       <p class="c-main-tagline">{{ main_tagline }}</p>
 
       <figure class="qr-code-wrapper">
-        <img src="{{ l10n_img('firefox/welcome/page17/qr-code.png') }}" width="200" height="200" alt="{{ qr_alt }}">
+        {{ qrcode('https://www.mozilla.org/firefox/browsers/mobile/app/?product=firefox&campaign=' + qr_campaign, 12) }}
```

Comment:
**Question:** Curious why we're adding a link here within the QR code instead of upading the image both here and in the WNP120 page? :)

Reply:
Only because in some places it was easier to generate an image directly like here, rather than save out another image. In other places we need an img, since it has the `data-mozillaonline-link` stuff built into it.